multimedia/VLC/files/patch-vda-ffmpeg34+.diff

```diff
@@ -1,5 +1,4 @@
-https://trac.macports.org/ticket/55164
```

Comment:
You may want to keep this.

Reply:
Did you forget to run `git add` or is it that you'd like to drop the ticket reference?

Reply:
Check again please, I can see the reference again from here.